src/app/core/utils/feature-toggle/feature-toggle.service.ts

```diff
@@ -49,4 +49,14 @@ export class FeatureToggleService {
       distinctUntilChanged()
     );
   }
+
+  /*
+   * Asynchronously returns a list of all features that contain the name.
+   */
+  getFeatures$(name: FeatureToggleType): Observable<FeatureToggleType[]> {
```

Comment:
this should be a selector in the configuration.selectors.ts file e.g. getFeaturesForType

Reply:
is obsolete and will reverted by next commit (inclusive regarding spec)